bamMergePreprocessing.wdl

```diff
@@ -467,37 +376,74 @@ task preprocessBam {
   }
 }
 
-task mergeBams {
+task filterBam {
   input {
-    Array[File] bams
+    Boolean doFilter = true
     String outputFileName
-    String suffix = ".merge"
-    String? additionalParams
 
-    Int jobMemory = 24
-    Int overhead = 6
+    # by default write tmp files to the current working directory (cromwell task directory)
+    # $TMPDIR is set by Cromwell
+    # $TMP is set by Univa
+    String temporaryWorkingDir = ""
+
+    File inputBam
+    File inputBamIndex
+    Array[String] intervals
+
+    # filter parameters
+    String filterSuffix = ".filtered"
+    Int filterFlags = 260
+    Int? minMapQuality
+    String? filterAdditionalParams 
+
+    String reference
+    Int jobMemory = 48
+    Int overhead = 8
     Int cores = 1
     Int timeout = 6
-    String modules = "gatk/4.1.6.0 python/2.7"
+    String modules = "samtools/1.9 gatk/4.1.6.0 python/2.7"
```

Comment:
Is it possible to update to `python/3.10.6`? Python 2.7 is EOL.

Reply:
Just found python oly used collectFilesBySample and indelRealign task, which have been removed from wdl, now we can remove python altogether.